.env.example

```diff
@@ -1,3 +1,8 @@
 DISABLE_FORCE_SSL=
 MIXPANEL_TOKEN=
-GA_TRACKINGID=
\ No newline at end of file
+GA_TRACKINGID=
+PHOTON_API_URL=https://st-photon-dev.herokuapp.com
```

Comment:
Let's leave the URL value out of this file. We can find a different and more private place to document any dev, staging and production values that can be used here.